modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -304,3 +304,17 @@ function acquia_cms_common_update_8009() {
     }
   }
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Set disabled FALSE to jsonapi resource for user to API menu.
+ */
+function acquia_cms_common_update_8010() {
```

Comment:
update hook number for this change to 8301 for 3.x release and 8201 for 2.x release(if releasing in 2.x)

Reply:
Done